pages/osx/sed.md

```diff
@@ -12,7 +12,7 @@
 
 - Replace all occurrences of a string in a file, overwriting the file (i.e. in-place):
```

Comment:
I'm keeping the ` (i.e. in-place)` because is in the Linux page too, but I don't like it.
Please highlight the `[i]` to get a meaning to that letter:
```suggestion
- Replace all occurrences of a string [i]n a file, overwriting the file (i.e. in-place):
```